setup.py

```diff
@@ -104,7 +104,7 @@ def run(self):
                         'ioflo==1.5.4',
                         'semver==2.7.9',
                         'base58==1.0.0',
-                        'orderedset==2.0',
+                        'orderedset==2.0.3',
```

Comment:
I believe it needs to be specified in `build-scripts/ubuntu-1604/build-3rd-parties.sh` as well (and in indy-node's repo too).

Reply:
Thanks for pointing out this

Reply:
Fixed